src/lib/ActionFactory.js

```diff
@@ -0,0 +1,27 @@
+import KeyMirror from 'keymirror'
+
+import ActionPrototype from './ActionPrototype'
+import AppDispatcher from './fu-dispatcher'
+import fetcher from './jsonFetcher'
+
+export default class ActionFactory {
+  //takes an array of 'strings which become action Types, via KeyMirror
```

Comment:
@andela-kadeniyi I'd like it if you would do some cleanup in this component: there should be a space after `//`, and there is a random single quote before `strings`